project-words.txt

```diff
@@ -1,19 +1,96 @@
+acqktp
+AESSBOX
+allclose
+arange
+argmax
+argmin
+argsort
+ascad
 astype
+autoreload
+batchnorm
+batchnormalization
+bibtex
+booktitle
+Bursztein
+burszteindc
+burzteindc
 bytelist
 chipwhisperer
+chipwispher
+clkgen
+cmap
+colorbar
+comparaison
```

Comment:
Same here. That's the French word for comparison :)

Reply:
Similar as above, part of function name at https://github.com/google/scaaml/blob/b01f2bb685938082b423a3f112aaa934c8da10f1/scaaml/plot.py#L68

My French vocabulary is growing :)